src/utils/utils.js

```diff
@@ -66,4 +66,20 @@ const releaseDates = [{ value: 'presence', label: 'Presence Of' }, { value: 'abs
 
 const curateFilters = [{ value: 'licensed', label: 'Licensed' }, { value: 'described', label: 'Described' }]
 
-export { setIfValue, difference, customLicenseIds, asObject, sorts, licenses, sources, releaseDates, curateFilters }
+const isMobile = window.innerWidth < 991
```

Comment:
the reference to "mobile" here feels overly specific. It is really "narrow". Could be narrow for lots of reasons, or indeed, wide on a mobile device.

Reply:
we could stick with the bootstrap 3 terminology

```
/* Extra small devices (phones, less than 768px) */
/* No media query since this is the default in Bootstrap */

/* Small devices (tablets, 768px and up) */
@media (min-width: @screen-sm-min) { ... }

/* Medium devices (desktops, 992px and up) */
@media (min-width: @screen-md-min) { ... }

/* Large devices (large desktops, 1200px and up) */
@media (min-width: @screen-lg-min) { ... }
```

from https://getbootstrap.com/docs/3.4/css/

Reply:
That sounds good. are we going to handle the multiple sizes or just <= small and >= medium? Recall my comment elsewhere here that my phone (recent-ish and clearly mobile) is 768 wide. So if we are going to do phones, feels like we have to target 768.

Reply:
Sounds good. I've fixed the CSS media queries